pennylane/math/single_dispatch.py

```diff
@@ -197,6 +197,7 @@ def _take_autograd(tensor, indices, axis=None):
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "arctan2"] = "tensorflow.math"
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "diag"] = "tensorflow.linalg"
 ar.autoray._SUBMODULE_ALIASES["tensorflow", "kron"] = "tensorflow.experimental.numpy"
+ar.autoray._SUBMODULE_ALIASES["tensorflow", "mod"] = "tensorflow.experimental.numpy"

```

Comment:
How does `tf.math.floormod` compare to `tf.experimental.numpy.mod`?

Reply:
The outputs are identical even for negative numbers, and it looks like `tf.math.floormod` is around 1.5x faster.